From FAIR software to Software in the Research Landscape (#121)
esciencecenter-digital-skills/research-software-support: modules/softwarelandscape/exercise3.md

---
title: Role playing
type: exercise
order: 4
---

> ## Exercise: Role playing exercise (15 minutes)

Reviewer: Nice, I think these role playing exercises are the right way to go for training support staff

Pick a piece of software you know of or pick one from the following examples:

- [ESMValTool](https://research-software-directory.org/software/esmvaltool)
- [LitStudy](https://research-software-directory.org/software/litstudy)
- [Haddock](https://research-software-directory.org/software/haddock3)
- [worcs](https://cjvanlissa.github.io/worcs/index.html)
- [democracy-topic-modelling](https://research-software-directory.org/software/democracy-topic-modelling)

> One person (the researcher) takes some time to prepare to presents a piece of software, the other person reads up on the [reading materials](reading.md) from this module. 

Reviewer: ```suggestion
One person (the researcher) takes some time to prepare to present a piece of software, while the other person reads up on the [reading materials](reading.md) from this module. 
```

?

Then you play out one of the following two situations:

> 1. A researcher comes to a supporter with their piece of software and asks for support on managing their piece of software.

Reviewer: ```suggestion
1. A researcher comes to a supporter with their piece of software and asks for support on managing it.
```

Maybe?

> 2. A research supporter approaches a researcher which is, according to the organization's records, contributer to the software.

Reviewer: ```suggestion
2. A research supporter approaches a researcher who is, according to the organization's records, a contributer to the software.
```
